### DISCUSSION TITLE
Bookmark

### DISCUSSION THREAD

**Discussion by joganaz** [electerm/electerm#3429] [Ideas]
Thank you for your really good job.

I am using a licensed instance of Royal TSX and I am migrating to Electerm.

I have some ideas for bookmark:
1. Unlimited inherited folders, or more than 2. If unlimited is not possible, 10 levels could be enough :-)
2. Notes as new type of bookmark element. Could be useful when you want to add some documentation to a bookmark folder
3. Option of remove type os connection in title. In web shows "[web]" in local "[local]" but in ssh it not shows nothing, so the option of you can hide "[web]", "[local]", ... from the title bookmark like ssh.
4. New VNC type 

Thank you, very much.

**Comment by zxdong262** [comment]
Thank you for the feedback, would do it in future releases
